CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Changed
+- Use latest version of conjur-authn-k8s-client which supports JWT loging and tracing.
+  [cyberark/secretless-broker#1446](https://github.com/cyberark/secretless-broker/pull/1446)
+
```

Comment:
I'm wondering if we should include a CHANGELOG.md entry to say that we bumped K8s API version to 1.23.
This *might* be considered a user-facing change, since it affects in which Kubernetes clusters the SP can be run.
(Although this may only affect users who are using CRD/CRs to configure the SP... since I think that's why we're depending on Kubernetes API client and machinery?).

Reply:
I believe it only affect CRDs